### ISSUE TITLE
Adabelief enabled causes model to spike and crash

### ISSUE THREAD

**Issue by kilerb** [iperov/DeepFaceLab#5278]
At least I believe it's adabelief that's causing the issue.  No crash problems prior to trying to use it.
I pretrained a new model with adabelief on (one thing to note is if I started with it on immediately it would crash...  So I had to pretrain with it off for a few iterations (200 or so) and then turn it on and then it pretrained to 300k no problem.
Then I started a new project with the new pretrained model.  Looked great for 2 days.  The graph did have some weird spikes in it, but other than that it was turning out great.  Woke up today and it was completely yellow/red in the right side of the training grid.  Below are my model settings and the training grid/graph once I noticed the crash this morning.  One other quick note, was comparing settings with another person that has had no problems with adabelief.  He's on a 3090...  I'm on an A6000...  Other than that, the only big difference in our environments is he's on linux and I'm on windows.  Also, I'm running off my NAS, not local.  Never had a problem with that in the past either.  So, not either of those factors would matter at all.  Anyway, here's the screenshots.  Thanks for any advice!!! :)

![image0](https://user-images.githubusercontent.com/6969681/107154722-69879180-6929-11eb-959a-a2cb40862dd0.png)
![image0](https://user-images.githubusercontent.com/6969681/107154723-6d1b1880-6929-11eb-947b-904ff3b13ff6.png)

**Comment by NvrTdp** [comment]
Have seen some agreement from other users that changing gradient clipping either way after you enable adabelief could get the model to collapse. I have not got any collapse yet, always had clip disabled. Last 600K on current model with ada enabled. 
Quite a lot of spikes, but nothing out of the ordinary other than that. 
![51638774-e50](https://user-images.githubusercontent.com/77741794/107218011-52bb6a80-6a0f-11eb-8f3a-28a2e086576a.jpg)

**Comment by ghost** [comment]
Did you ever find the answer?
